    OFrom 2305a0b0fe3e66e1cd95683af2c6f02e7ff29220 Mon Sep 17 00:00:00 2001
From: Antonio Prcela <antonio.prcela@gmail.com>
Date: Fri, 12 Mar 2021 09:40:40 +0100
Subject: [PATCH] Open links properly if ending with "

---
 src/fe-gtk/xtext.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)e

--

Modified by 0x5c: addded case for single quote character

diff --git a/src/fe-gtk/xtext.c b/src/fe-gtk/xtext.c
index 418bb4da5..fb4962e12 100644
--- a/src/fe-gtk/xtext.c
+++ b/src/fe-gtk/xtext.c
@@ -64,7 +64,7 @@
 
 /* is delimiter */
 #define is_del(c) \
-	(c == ' ' || c == '\n' || c == '>' || c == '<' || c == 0)
+	(c == ' ' || c == '\n' || c == '>' || c == '<' || c == 0 || c == '"' || c == '\'')
 
 /* force scrolling off */
 #define dontscroll(buf) (buf)->last_pixel_pos = 0x7fffffff
